kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -641,6 +641,380 @@ paths:
         404:
           description: Not Found
 
+  /api/clusters/{clusterName}/connectors:
```

Comment:
One cluster might be used by multiple instances of kafka connect, and we should be able to manage multiple from rest per cluster

Reply:
Updated, thanks